config.toml

```diff
@@ -295,4 +295,37 @@ weight = 4
 [[languages.fr.menus.main]]
 name = "Communauté"
 url = "https://github.com/helm/community"
-weight = 5
\ No newline at end of file
+weight = 5
+
+# Spanish
+[languages.es]
+title = "Helm"
+description = "Helm - El Administrador de Paquetes de Kubernetes."
+contentDir = "content/es"
+languageName = "Español"
```

Comment:
for consistency with the other languages, you could go with:

languageName = "Español Spanish"

(I agree this is not perfect and I suspect we may need to do some sorting soon.)